### ISSUE TITLE
Sketch color picker makes the Lexical text block lose focus when using the 

### ISSUE THREAD

**Issue by sunshineo** [casesandberg/react-color#901]
It works fine when using the buttons
![color-picker](https://github.com/casesandberg/react-color/assets/1072766/15c39df1-589b-4495-9c30-d82b0a6eeb2e)

**Comment by Nin0k** [comment]
@sunshineo Hello! Have you solved this problem?

**Comment by sunshineo** [comment]
Nope. Had to remember the selection and show a different highlight
